src/common/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import Link from "next/link";
+import { useRouter } from "next/router";
+
+import {
+  BookLineIcon,
+  DealsIcon,
+  StarLineAltIcon,
+} from "@/common/components/CustomIcon";
+import { Icon } from "@iconify-icon/react";
+import {
+  SidebarItem,
+  type SidebarListItem,
+} from "@/common/components/SidebarItem";
+import { Logo } from "@/common/components/Logo";
+import { Input } from "@/common/components/Input";
+
+export const Sidebar = () => {
+  const router = useRouter();
+
+  return (
+    <div className="h-full w-52 space-y-5 border-r border-r-border-elevated px-4 py-5">
+      <Link href="/" className="flex items-center px-3 text-primary-default">
+        <Logo />
+      </Link>
+      <Input
+        className="w-full"
+        leftContent={
+          <Icon icon="uil:search" size={16} className="px-1 text-text-em-low" />
+        }
+        placeholder="Search"
+        size={{ initial: "sm" }}
```

Comment:
Just FYI, if you wanna just use the prop, you can do so like this. We use the object way like you did here to make it responsive if needed! 

```suggestion
        size="sm"
```

Reply:
ah noted! thanks for the review @Aztriltus :)